docs/arastirma/uret.py

```
@@ -0,0 +1,405 @@
+}}
+.rb:hover {{ color:var(--accent); }}
+.rnum {{
+  font-family:"IBM Plex Mono", monospace; font-size:11px; color:var(--accent);
+  min-width:14px; font-weight:500;
+}}
+.yigin {{
+  margin:10px 0 14px 8px; padding:10px 0 10px 18px; position:relative;
+  border-left:1px solid var(--rule);
+}}
+.yigin::after {{
+  content:"bağımlılık yönü"; position:absolute; left:-1px; bottom:-19px;
```

Ruff (0.16.1)

[warning] String contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)